ipaserver/install/server/install.py

```diff
@@ -940,9 +940,24 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
     print("")
-    print("Next steps:")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("\nObtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
+    print("\nServer is installed with following roles :")
+    print("* Directory Server")
+    print("* Kerbeors Server")
+    print("* Httpd Server")
+    print("* CA Server")
+    print("* DNS Server")
+    print("\nFor troubleshooting and errors :")
+    print("Please visit : https://www.redhat.com/en")
```

Comment:
For FreeIPA, we should rather point users to ``freeipa.org`` and only RH IdM users to ``redhat.com``.